thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -1283,7 +1288,7 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
         }
         void processRows(CThorExpandingRowArray &processing, unsigned partNo, IKeyManager *keyManager)
         {
-            CStatsScopedThresholdDeltaUpdater scoped(activity.statsUpdater);
+            CStatsScopedThresholdDeltaUpdater scoped(statsUpdater);
```

Comment:
I think the conclusion from our offline discussion, was that this contextLogger capture/updating of inactiveStats should become redundant.
Because, you are now each handler has its own CStatsContextLogger, which the IKeyManager's are using, and recording jhtree stats to, and you are harvesting into 'fileStats'.

This process is still recording them into the activity stats (into inactiveStats), but I think we said it should not need to anymore, and instead they should be totaled (in the worker), from the collected fileStats.

However, it probably is worth keeping like this for this PR now, and removing the duplicated processes/code in a subsequent PR.